xilem/src/view/button.rs

```diff
@@ -20,39 +21,48 @@ pub struct Button<F> {
     callback: F,
 }
 
-impl<F, State, Action> MasonryView<State, Action> for Button<F>
+impl<F, State, Action> View<State, Action, ViewCtx> for Button<F>
 where
     F: Fn(&mut State) -> Action + Send + Sync + 'static,
 {
-    type Element = masonry::widget::Button;
+    type Element = Pod<widget::Button>;
     type ViewState = ();
 
-    fn build(&self, cx: &mut ViewCx) -> (WidgetPod<Self::Element>, Self::ViewState) {
-        cx.with_leaf_action_widget(|_| {
-            WidgetPod::new(masonry::widget::Button::new(self.label.clone()))
+    fn build(&self, ctx: &mut ViewCtx) -> (Self::Element, Self::ViewState) {
+        ctx.with_leaf_action_widget(|_| {
+            Pod::from(WidgetPod::new(widget::Button::new(self.label.clone())))
```

Comment:
This should use `.into` - it's a mistake I made due to the old signature of `ViewCtx::with_leaf_action_widget`